contrib/oci-seccomp-gen/syscall_compare.go

```diff
@@ -0,0 +1,145 @@
+package ociseccompgen
+
+import (
+	"fmt"
+	"reflect"
+	"strconv"
+	"strings"
+
+	types "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+/*******************************************************
+This file is a bunch of helper functions for determining
+what action should be taken for new syscalls.
+See github.com/grantseltzer/manhattan/Resources/Logic.png
+for a flowchart and explanation
```

Comment:
On Wed, Jul 06, 2016 at 07:00:14AM -0700, Grant Seltzer Richman wrote:
> I think it's a good idea, What do you think?

The flow-chart looks useful, but I'd like to have whatever vector
source you used to develop it alongside the raster PNG.